GC.cpp

```diff
@@ -209,7 +209,11 @@ void ObjectMemory::reclaimInaccessibleObjects(DWORD gcFlags)
 						nMaxUnmarked = 512;
 					else
 						nMaxUnmarked = nMaxUnmarked << 1;
+					OTE** oldPointer = pUnmarked;
```

Comment:
As I explained in my comments on [one of Bruno's PR's](https://github.com/dolphinsmalltalk/DolphinVM/pull/10), this change does not correctly address the issue, and is not worth making. In the extremely unlikely event that realloc fails, the program is going to crash anyway. 
I do not believe it worth trying to fix this code analysis issue.